debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImageProcess.java

```diff
@@ -578,24 +566,60 @@ public J9DDRBaseImageThread getThread(long id) {
 		Map<Long, Object> dtfjThreadMap = getThreadMap();
 		return (J9DDRBaseImageThread)dtfjThreadMap.get(id);
 	}
-	
-	private int resolveGenericSignal(int num) {
-		if ((num & J9PORT_SIG_FLAG_SIGQUIT) != 0) 	return 3;
-		if ((num & J9PORT_SIG_FLAG_SIGILL) != 0) 	return 4;
-		if ((num & J9PORT_SIG_FLAG_SIGTRAP) != 0) 	return 5;
-		if ((num & J9PORT_SIG_FLAG_SIGABRT) != 0) 	return 6;
-		if ((num & J9PORT_SIG_FLAG_SIGFPE) != 0) {
-			if (num == J9PORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO) 		return 35;
-			if (num == J9PORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO) 	return 36;
-			if (num == J9PORT_SIG_FLAG_SIGFPE_INT_OVERFLOW) 	return 37;
+
+	private static int resolveGenericSignal(int num) {
+		if (num == OMRPORT_SIG_FLAG_SIGQUIT) {
+			return 3;
+		}
+
+		if (num == OMRPORT_SIG_FLAG_SIGILL) {
+			return 4;
+		}
+
+		if (num == OMRPORT_SIG_FLAG_SIGTRAP) {
+			return 5;
+		}
+
+		if (num == OMRPORT_SIG_FLAG_SIGABRT) {
+			return 6;
+		}
+
+		if ((num & OMRPORT_SIG_FLAG_SIGFPE) != 0) {
+
+			if (num == OMRPORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO) {
+				return 35;
+			}
+
+			if (num == OMRPORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO) {
+				return 36;
+			}
+
+			if (num == OMRPORT_SIG_FLAG_SIGFPE_INT_OVERFLOW) {
+				return 37;
+			}
```

Comment:
A minor nit: these specific FPE checks need not be nested within the test for `OMRPORT_SIG_FLAG_SIGFPE`; they should be moved before line 587.

Reply:
Makes sense, new commit pushed